cucumber.js

```diff
@@ -2,10 +2,17 @@ require('dotenv').config();
 
 const worldParameters = {
   appUrl: process.env.BASE_APP_URL || 'http://localhost:4321',
+  generationServiceUrl: process.env.GENERATION_SERVICE_URL || 'http://localhost:4321',
+  authorizationHeader: process.env.AUTH_HEADER || null,
+  validatorServiceUrl: process.env.VALIDATOR_SERVICE_URL || null,
   mockPort: process.env.MOCK_PORT || null
 };
 
-const worldParametersLocal = { ...worldParameters, mockPort: 4321 };
+const worldParametersLocal = {
+  ...worldParameters,
+  validatorServiceUrl: 'http://localhost:4567/validate',
```

Comment:
Should this default to `null`? It was a bit confusing to me that the README states:

> Optional FHIR Validator Service url. If not provided, will skip validator test steps.

but then running the tests without spinning up the validator throws an error, because we can't connect to the validator service.

Reply:
worldParametersLocal are the config setting for the tests (probably should have a better name). So it overwrites the default null with localhost:4567, where the docker service is presumed to be running, for the mock tests. The docker-compose needs to be spun up in a separate step before running the tests.

Reply:
Okay, I didn’t realize the docker-compose is presumed to be running to run the tests. Should we add this as a comment somewhere in the README - I'm just thinking that the word "optional" next to "Run FHIR Validator Service" may be a bit confusing, since it isn’t actually optional if you are testing with the mock server. Also fine keeping as is if you don’t think it's necessary.

Reply:
I also think it would be useful to add a comment somewhere in the README about having docker-compose running before running the tests so that this step is more clear for the user.

Reply:
Updated test env to be configurable via .env.test file and added instructions to README.

Reply:
Looks good! Tests pass while the validator is running, and the validator test steps are properly skipped after removing `VALIDATOR_SERVICE_URL` from `.env.test`.